### DISCUSSION TITLE
About Windows OS version

### DISCUSSION THREAD

**Discussion by egenarcin** [rehlds/ReHLDS#947] [Q&A] [answered]
Hello,

The latest version "3.12.0.780" does not include ReHLDS for Windows OS. 

On one of the release patch notes, it is stated that Windows package removed temporarily.

Is it coming back soon?

**Comment by WaLkZa** [comment, marked as answer]
https://github.com/dreamstalker/rehlds/actions/runs/3083980652

**Reply by egenarcin** [reply]
Thanks.

**Reply by MathewG7** [reply]
Artifacts is expired

**Reply by stamepicmorg** [reply]
> Artifacts is expired

> it says it's expired

оpen actions, than click lstest build and you will see win32 build

**Comment by stamepicmorg** [comment]
there is no pure\non pure versions. is it merged to one unical?
